src/main/scala/midas/core/FPGATop.scala

```diff
@@ -166,11 +172,16 @@ class FPGATop(simIoType: SimWrapperIO)(implicit p: Parameters) extends Module wi
     }
   }
 
-  assert(dmaPorts.size <= 1)
-  if (dmaPorts.nonEmpty) {
+  if (dmaPorts.isEmpty) {
+    io.dma := DontCare
```

Comment:
Should you not tie off the appropriate ready and valids instead, or is this safe?

Reply:
I probably should tie them off, yeah.